File: leo2-node/src/test/java/org/deku/leo2/node/auth/IdentityTest.java
package org.deku.leo2.node.auth;

import org.apache.commons.logging.Log;
import org.apache.commons.logging.LogFactory;
import org.junit.Test;

/**
 * Created by masc on 27.06.15.
 */
public class IdentityTest {
    Log mLog = LogFactory.getLog(this.getClass());

    @Test
    public void testIdentity() {
        Identity ident = Identity.create(SystemInformation.create());
        mLog.info(ident);
    }
}

File: leoz-core/src/test/java/org/deku/leoz/IdentityTest.kt
package org.deku.leoz

import org.deku.leoz.bundle.BundleType
import org.deku.leoz.identity.DesktopIdentityFactory
import org.junit.Test
import org.junit.experimental.categories.Category
import org.slf4j.LoggerFactory
import sx.junit.StandardTest

/**
 * Created by masc on 27.06.15.
 */
@Category(StandardTest::class)
class IdentityTest {
    private var log = LoggerFactory.getLogger(this.javaClass)

    val identity by lazy {
        DesktopIdentityFactory(
                BundleType.LEOZ_NODE.value,
                SystemInformation.create()
        )
                .create()
    }

    @Test
    fun testIdentity() {
        log.info(this.identity.toString())
    }
}
